packages/venia-concept/src/components/PurchaseDetailsPage/OrderItem/ButtonGroup/__tests__/buttonGroup.spec.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ButtonGroup from '../buttonGroup';
+import Button from 'src/components/Button';
+
+configure({ adapter: new Adapter() });
+
+const classes = {
+    buttonGroupItem: 'buttonGroupItem',
+    textNodeContiner: 'textNodeContiner'
+};
+
+const onClickHandler1 = jest.fn();
+const textNode1 = 'text1';
+const onClickHandler2 = jest.fn();
+const textNode2 = 'text2';
+
+const buttonGroupItemsMock = [
+    {
+        onClickHandler: onClickHandler1,
+        iconName: 'icon',
+        textNode: textNode1
+    },
+    {
+        onClickHandler: onClickHandler2,
+        iconName: 'icon',
+        textNode: textNode2
+    }
+];
+
+test('renders all button group items correctly', () => {
+    const wrapper = shallow(
+        <ButtonGroup buttonGroupItems={buttonGroupItemsMock} />
+    ).dive();
+
+    expect(wrapper.find(Button)).toHaveLength(buttonGroupItemsMock.length);
+});
+
+test('clicking buttons triggers appropriate handlers', () => {
+    const wrapper = shallow(
+        <ButtonGroup
+            classes={classes}
+            buttonGroupItems={buttonGroupItemsMock}
+        />
+    ).dive();
+
+    wrapper
+        .findWhere(
```

Comment:
You can use forEach to handle each items in buttonGroupItems. Also you need to reset jest mock functions for all buttons.